barman/cli.py

```diff
@@ -790,6 +804,7 @@ def check(args):
     output.close_and_exit()
 
 
+@command()
 def diagnose():
```

Comment:
Just noticed this still needs a placeholder arg as argparse is going to give it a namespace.